Floria/HW5/2.py

```diff
@@ -0,0 +1,7 @@
+n = int(input("Enter a number: "))
+
```

Comment:
good job
but I would recommend use more meaningful name of variable
and look at this code
n = int(input('enter some number from fibonacci list= '))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])